lib/assist/constants.go

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2023 Gravitational, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package assist
+
+// MessageClasses contains type of assist message we expect users to send.
+// When running on Cloud we attempt to classify user messages in one of those
+// categories. If this succeeds, we send an event into the analytics pipeline.
+//
+// Keys are the category names, those are the ones reported in the event and
+// generated by the model. Values are the category description. They are used to
+// build the model prompt and allow to provide more context to the model to
+// improve the classification.
+var MessageClasses = map[string]string{
+	"command execution": "the user want to execute a command on one or many servers",
+	"troubleshooting":   "the user wants to diagnose a problem or understand an error message",
+	"configuration":     "the user wants to generate configuration for a software which is not Teleport",
+	"list resources":    "the user wants to list the available resources connected to the Teleport cluster",
```

Comment:
What if the user wants to add, describe or remove resources? Can we make this broader?